code/src/index.css

```diff
@@ -11,3 +11,74 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.date {
+  font-style: italic;
+}
+
+span {
+  margin: 0;
+  padding: 0;
+}
+
+.message-card,
```

Comment:
Good DRY-job styling two classes at once! =) But the background color in the design example was not the same in the form as in the cards...